packages/ui/src/components/wui-icon/index.ts

```diff
@@ -131,14 +133,15 @@ const svgOptions: Record<IconType, TemplateResult<2>> = {
   swapVertical: swapVerticalSvg,
   telegram: telegramSvg,
   twitch: twitchSvg,
-  twitter: twitterSvg,
+  twitter: xSvg,
   twitterIcon: twitterIconSvg,
   verify: verifySvg,
   verifyFilled: verifyFilledSvg,
   wallet: walletSvg,
   walletConnect: walletConnectSvg,
   walletPlaceholder: walletPlaceholderSvg,
-  warningCircle: warningCircleSvg
+  warningCircle: warningCircleSvg,
+  x: xSvg
```

Comment:
do we need both twitter and X?

Reply:
Magic still uses `twitter` as naming, so I think it is important to have both for now. We can adjust this later